spec/src/modules/autocomplete.js

```diff
@@ -525,7 +527,7 @@ describe('ConstructorIO - Autocomplete', () => {
       it('Should be rejected when network request timeout is provided and reached', () => {
         const { autocomplete } = new ConstructorIO(validOptions);
 
-        return expect(autocomplete.getAutocompleteResults(query, {}, {}, { timeout: 10 })).to.eventually.be.rejectedWith('The user aborted a request.');
+        return expect(autocomplete.getAutocompleteResults(query, {}, {}, { timeout: 10 })).to.eventually.be.rejectedWith('The operation was aborted.');
```

Comment:
The abort error message has changed in node-fetch 3. I don't believe it will cause any problems.